src/applications/appeals/996/tests/hlr.cypress.spec.js

```diff
@@ -6,21 +6,31 @@ import { createTestConfig } from 'platform/testing/e2e/cypress/support/form-test
 import formConfig from '../config/form';
 import manifest from '../manifest.json';
 import {
-  mockContestableIssues,
+  // mockContestableIssues,
```

Comment:
Should comment be deleted here?

Reply:
yeah, I forgot to delete this commented out line... not going to bother in this PR because CI takes so long to re-run

Reply:
I removed this anyway 😛